### PR TITLE
Chris validation 2

### PR DESCRIPTION
**Pull request by chrisroederucdenver**
is there stuff here to merge?

### CODE REVIEW

File: SimpleTerminologyLoadTest.ipynb [hunk 1]
```diff
@@ -0,0 +1,331 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "266cc6d5",
+   "metadata": {},
+   "source": [
+    "# Simple Terminology Load Test\n",
+    "this is wall-clock time where a server-side cache may help and give better results\n",
```
Comment (joeflack4) [inline SimpleTerminologyLoadTest.ipynb:9]: @chrisroederucdenver What is the main difference between this one and #7? I see it's the same files added/edited. I see the same kind of cells as well, but I may not have looked closely enough.

Comment (chrisroederucdenver) [reply SimpleTerminologyLoadTest.ipynb:9]: Note the comment. I wasn't sure validation-2 has made it into the source base, and creating a PR is a lazy way of using github to show the differences. It was almost 8:00pm (late for me). Thanks for looking and understanding. :-)

Comment (chrisroederucdenver) [reply SimpleTerminologyLoadTest.ipynb:9]: I suppose I could accept this one first, and then see if github changes the diffs much on the other one.

Comment (joeflack4) [reply SimpleTerminologyLoadTest.ipynb:9]: Ahhh I see. Yeah, I suppose that'd be one way to do it. Oof, I guess that's even rougher than reviewing Jupyter code in a diff; trying to resolve a merge conflict. GL.

Comment (chrisroederucdenver) [reply SimpleTerminologyLoadTest.ipynb:9]: GL "good luck" LOL! yeah. That'll teach me!!

**Pull request merged**